packages/dev/serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1232,88 +1304,23 @@ export class GLTFExporter {
                         this._nodesCameraMap.get(gltfCamera)?.push(parentNode);
                         return null; // Skip exporting this node
                     }
-                } else {
-                    if (state.convertToRightHanded) {
-                        ConvertToRightHandedNode(node);
-                        RotateNode180Y(node);
-                    }
-                    this._nodesCameraMap.get(gltfCamera)?.push(node);
                 }
+                if (state.convertToRightHanded) {
+                    ConvertToRightHandedNode(node);
+                    RotateNode180Y(node);
+                }
+                this._nodesCameraMap.get(gltfCamera)?.push(node);

```

Comment:
This logic is different than before since it's outside of the else statement. Is this intentional?

Reply:
It was intentional. If the original `if` clause succeeds, it early returns. If it doesn't succeed (i.e., the parent exists, was added by importer, but it was skipped for export), I figured we should continue as normal.